ui/src/layouts/file_preview.rs

```diff
@@ -210,9 +190,9 @@ fn resize_window(
     Some(())
 }
 
-fn update_theme_colors(state: UseSharedState<State>) -> String {
+fn update_theme_colors() -> String {
+    let state = State::load();
```

Comment:
Down the road it may be best to avoid loading directly from the state since there could be many things that could trigger a chance and possibly cause a race in some way with how data is handled in memory vs how its stored then loaded from disk.